tool/travis_macos.sh

```diff
@@ -0,0 +1,253 @@
+    build_conan_and_upload "gRPC/1.8.3@inexorgame/stable"
+    build_conan_and_upload "libpng/1.6.34@bincrafters/stable"
+    build_conan_and_upload "Boost/1.66.0@conan/stable"
+    build_conan_and_upload "fmt/4.1.0@bincrafters/stable"
+    build_conan_and_upload "spdlog/0.14.0@bincrafters/stable"
+
+    # build_conan_and_upload "cmake-findboost/0.2.0@bincrafters/stable"
+    # build_conan_and_upload "boost_filesystem/1.66.0@bincrafters/testing"
+    # build_conan_and_upload "boost_program_options/1.66.0@bincrafters/testing"
+    # build_conan_and_upload "boost_random/1.66.0@bincrafters/testing"
+    # build_conan_and_upload "boost_regex/1.66.0@bincrafters/testing"
+    # build_conan_and_upload "boost_thread/1.66.0@bincrafters/testing"
+
+    build_conan_and_upload "InexorGlueGen/0.6.3@inexorgame/stable"
+
+    build_conan_and_upload "libjpeg-turbo/1.5.2@bincrafters/stable"
+    build_conan "SDL2/2.0.5@lasote/testing"
+    build_conan "SDL2_image/2.0.1@lasote/stable"
+
+    build_conan "doxygen/1.8.13@inexorgame/stable"
+    build_conan "CEF/3.3239.1709.g093cae4@inexorgame/testing"
+
+    conan info .
+
+    if test "$NIGHTLY" = conan; then
+      echo "execut conan install . --env build_all=1 --build -s compiler.libcxx=libc++"
+      conan install . --env build_all=1 --build -s compiler.libcxx="libc++"
+    else
+      echo "execut conan install . --env build_all=1 --env create_package=1 --build=missing -s compiler.libcxx=libc++"
+      conan install . --env build_all=1 --env create_package=1 --build=missing -s compiler.libcxx="libc++"
+    fi
+
+    conan build "$gitroot"
+
+    # Moving the CPack package to the /inexor directory, so we are able to access it from outside of Docker
+    local tempdir="/tmp/inexor-build/"
+    local zipname="inexor-core-${last_tag}-Linux64.zip"
+    local outputdir="/inexor/build/cpack/"
+    mkdir -pv ${outputdir}
+    mv -f -v -u "${tempdir}${zipname}" "${outputdir}"
+  )
+}
+
+run_tests() {
+  if contains "$TARGET" macos; then
+    "${bin}/unit_tests.exe"
+  else
+    echo >&2 "ERROR: UNKNOWN TRAVIS TARGET: ${TARGET}"
+    exit 23
+  fi
+}
+
+
+# ATTENTION:
+# Please USE the following naming format for any files uploaded to our distribution server
+# <BRANCHNAME>-<BUILDNUMBER>-<TARGET_NAME>.EXTENSION
+# where <PACKAGENAME> is NOT CONTAINING any -
+# correct: master-1043.2-linux_gcc.txt
+# correct: refactor-992.2-apidoc.hip
+# exception: master-latest-<TARGET_NAME>.zip
+# wrong: ...-linux_gcc-1043.2.zip
+
+## UPLOADING BUILDS AND THE APIDOC #################
+
+# upload remote_path local_path [local_paths]...
+#
+# Upload one or more files to our nightly or dependencies server
+# Variables are defined on the Travis website
+upload() {
+  # Fix an issue where upload directory gets specified by subsequent upload() calls
+  ncftpput -R -v -u "$FTP_USER" -p "$FTP_PASSWORD" "$FTP_DOMAIN" /macos/ "$@" || true
+}
+
+
+## increment the version number based on the last tag.
+incremented_version()
```

Comment:
you could leave this incremented_version and stuff out, but I assume you just copied it, so its fine for now.
But we need to remove the clutter later.

Reply:
Sure, this is heavily work in progress.